packages/ui/src/print-content/PrintContent.css

```diff
@@ -65,12 +64,12 @@
   .ui-Header,
   .web-Walkthrough--StepTracker,
   .web-WalkthroughFooter,
-  .u-container-walkthrough,
+  .p-hide,
   .ui-PdfResultDownload-container,
   .ui-PdfResultDownload-button,
   .ui-PdfResultDownload-imageContainer,
   .ui-PdfResultDownload-aside {
-    display: none;
+    display: none !important;
```

Comment:
Why is important needed?

Reply:
Because I added the utility hide class to hide the extra items needed on the result screen. Since we're hiding all these for printing we definitely want to make sure they're all always hidden.